src/AC/Deal.php

```diff
@@ -0,0 +1,128 @@
+<?php
+namespace AC;
+
+class Deal extends ActiveCampaign {
+	
+	function add($params, $post_data) {
+		$request_url = "{$this->url}&api_action=deal_add&api_output={$this->output}";
+		$response = $this->curl($request_url, $post_data);
+		return $response;
+	}
+
+	function edit($params, $post_data) {
```

Comment:
AFAIK, the arguments can be type-hinted to enforce `array` arguments to be passed